test/foundry/dapp/Application.t.sol

```diff
@@ -523,15 +701,38 @@ contract ApplicationTest is ERC165Test {
         _appOwner = _newAddr();
         _recipient = _newAddr();
         _tokenOwner = _newAddr();
+        _ens = ENS(_newAddr());
+        _resolver = AddrResolver(_newAddr());
         _interfaceIds.push(type(IApplication).interfaceId);
         _interfaceIds.push(type(IERC721Receiver).interfaceId);
         _interfaceIds.push(type(IERC1155Receiver).interfaceId);
+        for (uint256 i; i < 7; ++i) {
+            _tokenIds.push(i);
+            _initialSupplies.push(_initialSupply);
+            _transferAmounts.push(
+                bound(uint256(keccak256(abi.encode(i))), 0, _initialSupply)
```

Comment:
I prefer keeping it this way, because:
1. the function doesn't get used elsewhere;
2. by keeping it in the current place, it's easier to see what's the range of the random amount, right?

Reply:
Fair enough. :-)